QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java

```diff
@@ -59,9 +60,9 @@ public void autonomousMain() throws AutonomousModeOverException, InterruptedExce
                     Rollers.closed.set(closed);
                 }
             } finally {
-				Elevator.setBottom.event();
+		Elevator.setBottom.event();
```

Comment:
If you're going to fix the indentation here, remember that we use four spaces per indent.